### PR TITLE
fix(*) passthrough on a Mesh update

### PR DESCRIPTION
**Pull request by nickolaev**
Signed-off-by: Nikolay Nikolaev <nikolay.nikolaev@konghq.com>

### CODE REVIEW

File: docs/docs/0.7.2/policies/mesh.md [hunk 1]
```diff
@@ -120,3 +120,33 @@ mesh: default # indicate to Kuma what is the Mesh that the resource belongs to
 ```
 :::
 ::::
+
+### Controlling the passthrough mode
+
+The passtrhough mode is enabled by default on all the dataplanes in transparent proxy mode in a Mesh. This behavior can be changed by setting the `networking.outbound.passtrhough` in the Mesh resource. Example:
```
Comment (austince) [inline docs/docs/0.7.2/policies/mesh.md:126]:
```suggestion
The passthrough mode is enabled by default on all the dataplanes in transparent proxy mode in a Mesh. This behavior can be changed by setting the `networking.outbound.passthrough` in the Mesh resource. Example:
```

File: docs/docs/0.7.2/policies/mesh.md [hunk 2]
```diff
@@ -120,3 +120,33 @@ mesh: default # indicate to Kuma what is the Mesh that the resource belongs to
...
+type: Mesh
+name: default
+networking:
+  outbound:
+    passthrough: false
+```
+:::
+::::
+
+When `networking.outbound.passtrhough` is `false`, no traffic to any non-meh resource can leave the Mesh.
```
Comment (austince) [inline docs/docs/0.7.2/policies/mesh.md:152]:
```suggestion
When `networking.outbound.passthrough` is `false`, no traffic to any non-mesh resource can leave the Mesh.
```

**Review by austince — CHANGES_REQUESTED** [review]
Small typos

**Review by austince — APPROVED** [review]

**Comment by subnetmarco** [comment]
@nickolaev this should be against `next` not `master`.

**Pull request merged**